WORKSPACE

```
@@ -16,8 +16,8 @@ rules_python_version = "0.8.1"
 http_archive(
     name = "rules_python",
     sha256 = "cdf6b84084aad8f10bf20b46b77cb48d83c319ebe6458a18e9d2cebf57807cdd",
-    strip_prefix = "rules_python-%s" % rules_python_version,
-    url = "https://github.com/bazelbuild/rules_python/archive/refs/tags/%s.tar.gz" % rules_python_version,
+    strip_prefix = "rules_python-{0}".format(rules_python_version),
+    url = "https://github.com/bazelbuild/rules_python/archive/refs/tags/{0}.tar.gz".format(rules_python_version),
```

Would using f-strings be an option? If yes, this seems like this would be both shorter and less error prone, as token and values are not kept separate.
```suggestion
    url = f"https://github.com/bazelbuild/rules_python/archive/refs/tags/{rules_python_version}.tar.gz",
```
Only limitation is in the rare (only once here) situations where the same value is used more than once.
Haven't tested if our Bazels' Python version supports it, and I might be missing something else too.

---

Bazel doesn't have support for it. https://github.com/bazelbuild/starlark/issues/91